### PR TITLE
Update availability-replica-is-disconnected.md

### PR DESCRIPTION
**Pull request by treytMSFT**
Fixing typo "naode" to "node".
Changing Test-Netconnection to more clearly show it should be used to connect to the remote sql node instead of local.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@MashaMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ The following are possible solutions for this issue:
    Use the following commands to test connectivity in both directions from Node1 to Node2 and Node2 to Node1:
 
    ```PowerShell
-   $computer = $env:computername
+   $computer = "remote_node" 	# Replace with Node name as per your environment.
```
Comment (rwestMSFT) [inline docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md]:
```suggestion
   $computer = "remote_node" 	 # Replace with node name in your environment.
```

File: docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ The following are possible solutions for this issue:
    1. Then you can test the connection to the remote node. For example:
 
       ```PowerShell
-      $computer = "remote_node" # Replace with Naode name as per your environment.
+      $computer = "remote_node" # Replace with Node name as per your environment.
```
Comment (rwestMSFT) [inline docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md]:
```suggestion
      $computer = "remote_node" # Replace with node name in your environment.
```

File: docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ The following are possible solutions for this issue:
    Use the following commands to test connectivity in both directions from Node1 to Node2 and Node2 to Node1:
 
    ```PowerShell
-   $computer = $env:computername
+   $computer = "remote_node" 	 # Replace with node name in your environment.
```
Comment (rwestMSFT) [inline docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md]:
```suggestion
   $computer = "remote_node" 	  # Replace with node name in your environment.
```

**Pull request closed without merging**